src/components/ebay-combobox/component.ts

```diff
@@ -233,7 +226,7 @@ export default class Combobox extends Marko.Component<Input, State> {
         this.lastValue = input.value;
         this.state = {
             currentValue: this.lastValue,
-            viewAllOptions: (this.state && this.state.viewAllOptions) || true,
+            viewAllOptions:  input.viewAllOptions ?? true,
```

Comment:
Should the default value be `false` instead of true ?

Reply:
Actually, it looks like `state.viewAllOptions` is never changed? If it's not a mutable value we should just remove it from state and replace all instances of `state.viewAllOptions` with `(input.viewAllOptions ?? true)`

Reply:
Im almost of the opinion that the logic should stay the same, only the input should determine if the viewAllOptions is set to true or false. 
I don't think we should be setting this to true/false onInput. 
I think when we set `viewAllOptions` in the code, we should check the input version of it and then set it based on that.